checkpoint/latest_version.go

```diff
@@ -35,6 +35,10 @@ type LatestVersion struct {
 	SkipChecksumVerification bool
 	InstallDir               string
 
+	// LicenseDir represents directory path where to install license files (required for enterprise versions, optional for OSS)
```

Comment:
```suggestion
	// LicenseDir represents directory path where to install license files.
```
Since enterprise versions of products cannot be installed via Checkpoint and relatedly there is no `Enterprise` field in this struct I think we can leave this explanation out.